packages/high-roller-bot/src/utils.ts

```diff
@@ -49,22 +50,29 @@ export async function fetchMultisig(baseURL: string, token: string) {
   return (await getUser(baseURL, token)).multisigAddress;
 }
 
-// FIXME: cross-reference addresses to named-users so we can avoid using
-// Alice and Bob, especially when looking at logs
+/// Deposit and wait for counterparty deposit
 export async function deposit(
   node: Node,
   amount: string,
-  multisigAddress: string,
-  skip: boolean = false
+  multisigAddress: string
 ) {
-  if (skip) {
-    return;
+  const myFreeBalanceAddress = fromExtendedKey(
+    node.publicIdentifier
+  ).derivePath("0").address;
```

Comment:
I think the `node` object has a method to retrieve this, no? If not, we should add it.